sr_utilities_common/src/sr_utilities_common/heartbeat_publisher_mock.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import rospy
+import rospkg
+import sys
+import termios
+import tty
+from std_msgs.msg import Bool
+from threading import Thread, Lock
+
+
+class HeartbeatPublisherMock(object):
+    def __init__(self):
+        self._threads_running = None
+        self._input_thread = None
+        self._heartbeat_status = False
+
+        self._threading_lock = Lock()
+        self._teleop_heartbeat_publisher = rospy.Publisher('/teleop_heartbeat', Bool, queue_size=10)
```

Comment:
please parametrize topic name as contructor param. Also, default should have "teleop" in the name, we might use it for many different purposes